### PR TITLE
[FEAT] Fiat event tickets

### LINKED ISSUE #1: Ticket Price extension, Quantity & Description
Hello LNbits Team,

The ticket option is used more and more and also by organizers who are not Bitcoiner. Therefore, it would be great if the price can also be specified in fiat.

The function already exists with the extensions LN-Device and Marketplace etc.

This is still missing for the tickets. Likewise a ticket quantity, if one would like to take along the grandmother and the uncle.

Furthermore, it would be nice if the description is formatable. So the organizer can show his details like time, ticket price, place, short info, nicely and it is not a big text block.

Thank you for the great work

### PR DESCRIPTION
**Pull request by talvasconcelos**
Address requests made on #1, apart from the multiple ticket buy.

- Events can now be created denominating tickets in fiat
- Event description can be made in `Markdown`
- Ticket buy page shows an html parsed version of the `Markdown` for the description
- Fix a small bug when clicking the buy button
- Added the extra fields on payment for CSV export

event dialog
description

Closes #1

### CODE REVIEW

**Comment by dni** [comment]
@talvasconcelos rebase

**Review by dni — CHANGES_REQUESTED** [review]
rebase

**Comment by talvasconcelos** [comment]
These rebases always kill me!!

**Comment by talvasconcelos** [comment]
It's failing now... needs to be tested and fixed! I'll remove it from draft when it's ready

**Comment by arbadacarbaYK** [comment]
pls also change readme on merge

**Comment by talvasconcelos** [comment]
@arbadacarbaYK what do you mean? What needs change? Logic is still the same

**Comment by arbadacarbaYK** [comment]
> @arbadacarbaYK what do you mean? What needs change? Logic is still the same

Screenshots and description :)

File: templates/events/display.html [hunk 1]
```diff
@@ -85,7 +90,13 @@ <h5 class="q-my-none">{{ event_info }}</h5>
 </div>
 
 {% endblock %} {% block scripts %}
+<script src="https://unpkg.com/showdown/dist/showdown.min.js"></script>
```
Comment (dni) [inline templates/events/display.html:93]: i dont like the fact that we add it via cdn.
maybe we should add it to core via npm aswell, and use it on our `index.html` aswell

**Comment by bitkarrot** [comment]
The addition of markdown here is great! Looking good @talvasconcelos

**Comment by talvasconcelos** [comment]
__Needs PR lnbits/lnbits#1956 to be merged on LNbits first__

File: templates/events/display.html (hunk 1 above)
Comment (talvasconcelos) [reply templates/events/display.html:60]: Check lnbits/lnbits#1956

Comment (dni) [reply templates/events/display.html]: remove it already :)

**Comment by DoktorShift** [comment]
Nice topic. Fiat denomination here would be great since it's easier for merchants which are not into/familiar with bitcoin accounting but want to use this and get comfortable with it.

File: templates/events/display.html (hunk 1 above)
Comment (dni) [reply templates/events/display.html]: merged

**Comment by dni** [comment]
required pr is mergend and release in `0.11.0` @talvasconcelos  bump!

**Comment by talvasconcelos** [comment]
@dni please unlock and merge?

**Review by dni — APPROVED** [review]

**Pull request merged**